event_app.py

```diff
@@ -374,18 +386,18 @@ def processes():
 
 @app.errorhandler(HTTPRequestError)
 def handle_bad_request_error(e):
-    get_logger().error(str(e))
+    app.logger.error(str(e))
     return jsonify(error=str(e)), e.status_code
 
 
 @app.errorhandler(EventNotFoundError)
 def handle_event_not_found_error(e):
-    get_logger().error(str(e))
+    app.logger.error(str(e))
     return jsonify(error=str(e)), 404
 
 
 @app.errorhandler(ProcessNotFoundError)
 def handle_process_not_found_error(e):
-    get_logger().error(str(e))
+    app.logger.error(str(e))
     return jsonify(error=str(e)), 404
```

Comment:
Not direcly related to this PR. Couldn't we use some kind of requests package constant for 404 code ? (and may turn it into 400 maybe)

Reply:
Not sure what do you mean with "request package constant for 404 code". The `app.errorhandler` decorator is the Flask facility to handle exceptions.

I don't remember why it was set to return `404` I agree it should be `400`

Reply:
Actually `404` is correct as the exception is handling a case where resources are not found, not a bad request.